.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
```

Comment:
Pretty much everything in .idea needs to be .gitignored since it's IDE generated and everyone is gonna change them all the time.